controller/src/main/java/io/pravega/controller/store/stream/PersistentStreamBase.java

```diff
@@ -500,6 +562,18 @@ public String getScopeName() {
     }
 
     @Override
+    public CompletableFuture<Pair<Integer, List<Integer>>> getLatestEpoch() {
+        // TODO: this implementation needs to change once we do epoch change in history table
```

Comment:
I suppose these TODOs don't have issues associated because they are going to be fixed in the next PRs?